### DISCUSSION TITLE
Please add docker-compose example and document example.

### DISCUSSION THREAD

**Discussion by KleinByte** [nicolargo/glances#2154] [General]
Docker-compose should be the main form of initializing any docker project. And should be presented as such.
Running a docker container from the command line is not proper. 

Requesting that Glances adds a docker compose example to their docker install documentation.

**Comment by nicolargo** [comment]
A simple example is already available here: https://github.com/nicolargo/glances/blob/develop/docker-compose/docker-compose.yml
